docs/examples/button/custom.vue

```diff
@@ -0,0 +1,6 @@
+<template>
+  <el-row>
+    <el-button color="#626aef" style="color: white">Cutsom</el-button>
```

Comment:
This demo is actually a little bit confusing for me, since you have passed `color` as props instead you set `style="color: white"` which one would take effect?